### DISCUSSION TITLE
CLI Unknown Option -wt on Windows 11

### DISCUSSION THREAD

**Discussion by sford01** [chidiwilliams/buzz#992] [General]
Using CLI on Windows 11, when calling Buzz.exe with -wt or  --word-timestamps, I see a message box "Buzz: Unknown option 'wt'" or "Buzz: Unknown option 'word-timestamps". Buzz v 1.1.0. 

Could someone clarify the correct syntax or command please? Thanks!

**Comment by raivisdejus** [comment]
This feature is currently only available in the latest development version 1.2.0.

1. Go to https://github.com/chidiwilliams/buzz/actions/workflows/ci.yml?query=branch%3Amain
2. Take latest build and scroll down to the artifacts section
3. Download installation files. Note, you need to be logged in the Github to see the download button.

**Reply by sford01** [reply]
Apologies, I'm not seeing the option from your link to "take latest build and scroll down to the artifacts section". I've attached a screenshot of what I see from your link. I am logged in to Github.
<img width="1440" alt="Screenshot 2024-11-17 at 14 16 10" src="https://github.com/user-attachments/assets/7367c11e-7836-4e2a-89cd-48edc6d6d6ac">
Your help is very much appreciated!

**Reply by raivisdejus** [reply]
@sford01 Click on the first link `Adding support for cookiefile for Youtube downloads (#988)` ans scroll down to the bottom of the page

**Reply by sford01** [reply]
@raivisdejus thank you for your help, I have downloaded, extracted and completed the setup, but unfortunately see the following when attempting to run on Windows 11: 
![Screenshot 2024-11-17 153651](https://github.com/user-attachments/assets/37de99c2-b4cf-47ee-a803-f139125d230e).

**Reply by raivisdejus** [reply]
@sford01 Please uninstall the Buzz and then install it again. Some part of the old installation may be left and interfering with the new version. So clean re-installation may help this.

**Reply by sford01** [reply]
@raivisdejus Thank you, I did as instructed. 
I can confirm that using Whisper Tiny still causes a similar crash, however using faster whisper (tiny) works.
So there is still some instability, but we have a solution of sorts.
Thanks for your help!

**Comment by raivisdejus** [comment]
Issue with CLI is fixed in the latest version https://github.com/chidiwilliams/buzz/releases
